provider/k8s/template.go

```diff
@@ -126,9 +129,45 @@ func (p *Provider) templateHelpers() template.FuncMap {
 		"volumeTo": func(v string) (string, error) {
 			return volumeTo(v)
 		},
+		"splitIntoLines": splitIntoLines,
 	}
 }
 
+func splitIntoLines(value interface{}) []string {
```

Comment:
why this is necessary?

Reply:
To solve issue of templating the value of interface type, I created function to split the interface into []string, and then traverse over it and template that.